File: cost.java
//calculates total cost for one item
import java.util.Scanner;

public class FoodCostCalculator {
  public static void main(String[] args) {
    // Create a new Scanner object to read input from the user
    Scanner scanner = new Scanner(System.in);

    // Prompt the user for the name of the food item
    System.out.print("Enter the name of the food item: ");
    String foodItem = scanner.nextLine();

    // Prompt the user for the number of items
    System.out.print("Enter the number of items: ");
    int numItems = scanner.nextInt();

    // Prompt the user for the cost per item
    System.out.print("Enter the cost per item: ");
    double costPerItem = scanner.nextDouble();

    // Calculate the total cost
    double totalCost = numItems * costPerItem;

    // Print the total cost
    System.out.println("The total cost for " + numItems + " " + foodItem + " is $" + totalCost);
  }
}
import java.util.Scanner;

public class FoodCostCalculator {
  public static void main(String[] args) {
    // Create a new Scanner object to read input from the user
    Scanner scanner = new Scanner(System.in);

    // Prompt the user for the name of the food item
    System.out.print("Enter the name of the food item: ");
    String foodItem = scanner.nextLine();

    // Prompt the user for the number of items
    System.out.print("Enter the number of items: ");
    int numItems = scanner.nextInt();

    // Prompt the user for the cost per item
    System.out.print("Enter the cost per item: ");
    double costPerItem = scanner.nextDouble();

    // Calculate the total cost
    double totalCost = numItems * costPerItem;

    // Print the total cost
    System.out.println("The total cost for " + numItems + " " + foodItem + " is $" + totalCost);
  }
}

File: cost.py
# define a function that calculates the total cost of a purchase
def calculate_total_cost(food_type, number_of_items, cost_per_item):
    total_cost = number_of_items * cost_per_item
    return total_cost


# get input from the user
food_type = input("Enter the type of food: ")
number_of_items = input("Enter the number of items: ")
cost_per_item = input("Enter the cost per item: ")

# convert the inputs to numbers (floats)
try:
    number_of_items = float(number_of_items)
    cost_per_item = float(cost_per_item)
except ValueError:
    print("Invalid input. Please enter valid numbers.")

# calculate the total cost
total_cost = calculate_total_cost(food_type, number_of_items, cost_per_item)

# print the result
print("The total cost of your purchase is: $" + str(total_cost))
